src/ctia/entity/incident.clj

```diff
@@ -149,7 +149,8 @@
       :categories em/token
       :discovery_method em/token
       :intended_effect em/token
-      :assignees em/token})}})
+      :assignees em/token
+      :promotion_method em/token})}})
```

Comment:
This require an update of the mapping from OPS, unless it will be ignored and not indexed (and thus not searchable).